src/message_pack/token.cr

```diff
@@ -30,15 +49,13 @@ class MessagePack::Token
 
   def to_s(io)
     case @type
-    when :nil
-      io << :nil
```

Comment:
Wasn't sure if I should remove this or not.  As this code basically just print the type symbol as a string, that is handled as part of the else case which also does the same thing.